model/patch/patch.go

```diff
@@ -665,6 +665,7 @@ func (p *Patch) SetFinalized(ctx context.Context, versionId string) error {
 	p.Activated = true
 	p.ProjectStorageMethod = ""
 	p.PatchedProjectConfig = ""
+	p.LocalModuleIncludes = nil
```

Comment:
I would recommend writing a test for FinalizePatch that utilizes these changes so you can verify this change and the merging change, rather than just testing manually.

Reply:
added a test for load project into because finalize patch tests doesn't test the local modules functionality

Reply:
does not add a db key but instead opts to not save anything in the DB since  we don't need to

Reply:
Maybe no need to unset this here then

Reply:
it wouldn't really change anything at this point so I'll just delete it